packages/components-js/tests/vrt/puppeteer/helpers/puppeteer-helper.ts

```diff
@@ -23,7 +26,7 @@ export const setContentWithDesignSystem = async (page: Page, content: string, op
         <base href="http://localhost:8575"> <!-- NOTE: we need a base tag so that document.baseURI returns something else than "about:blank" -->
         <script type="text/javascript" src="http://localhost:8575/index.js"></script>
         <link rel="stylesheet" href="assets/styles.css" />
-        ${options.injectIntoHead}
+        ${initialStyles.concat(options.injectIntoHead)}
```

Comment:
I'd probably remove the concat.  
Also shouldn't the styles be applied before `assets/styles.css` to be in sync with other VRTs of static pages?